<?php

/*
|
|--------------------------------------------------
| CoreException test class
|--------------------------------------------------
|
| Test decription coming soon
|
*/

namespace Exceptions;

class CoreException
{
  public function __construct()
  {
    echo self::class . ' Успешно подключен<br />';
  }
}
